controllers/cmd/start.go

```diff
@@ -41,6 +42,20 @@ type ArgoEventsControllerOpts struct {
 	HealthPort       int32
 }
 
+func lookupEnvDurationOr(key string, o time.Duration) *time.Duration {
+	logger := logging.NewArgoEventsLogger().Named(eventbus.ControllerName)
+	v, found := os.LookupEnv(key)
+	if found && v != "" {
+		d, err := time.ParseDuration(v)
+		if err != nil {
+			logger.With(key, v).Fatalw("failed to parse", zap.Error(err))
```

Comment:
```suggestion
			logger.With(key, v).Fatalw("parsing %s duration", key, zap.Error(err))
```

We already know it's a fatal error so no need to add failed, maybe more info like the suggestion would be more useful.

Reply:
Updated!